internal/perms/grants_test.go

```diff
@@ -1072,6 +1072,31 @@ func Test_Parse(t *testing.T) {
 	}
 }
 
+// Fuzz_Parse is focused on finding panics
+func Fuzz_Parse(f *testing.F) {
+	tc := []string{
+		"id=*;type=*;actions=read,list;output_fields=",
+		`{"id": "*", "type": "*", "actions": ["read", "list"], "output_fields": []}`,
+		"id=*;type=*;actions=read,list",
+		"ids=*;type=*;actions=read,list",
+		`{"type":"host-catalog","actions":["create"]}`,
+		`{"id":"u_foobar","actions":["read"]}`,
+		`{"id":"u_foobar","actions":["read"],"output_fields":["version","id","name"]}`,
+		`{"id":"u_foobar","output_fields":["version","id","name"]}`,
+		`type=host-catalog;actions=create`,
+		`ids=hcst_foobar,hcst_foobaz;actions=read;output_fields=version,ids,name`,
+		`id=hcst_foobar;actions=read`,
+		`id=acctpw_foobar;actions=read`,
+		`id={{    user.id}};actions=read,update`,
+	}
+	for _, tc := range tc {
+		f.Add(tc)
+	}
+	f.Fuzz(func(t *testing.T, s string) {
+		_, _ = Parse(context.Background(), "global", s)
+	})
+}
+
```

Comment:
There's already a parse fuzz test at the bottom of this file, why don't we add these cases to that?

Reply:
I'll add these tests to the other one and run that one in CI